Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs

```diff
@@ -203,13 +200,17 @@ private void OnVictimInfectedMapInit(Entity<VictimInfectedComponent> victim, ref
 
     private void OnVictimInfectedRemoved(Entity<VictimInfectedComponent> victim, ref ComponentRemove args)
     {
-        _blindable.UpdateIsBlind(victim.Owner);
+        if (_status.HasStatusEffect(victim, "Muted", null) && _status.HasStatusEffect(victim, "TemporaryBlindness", null))
+        {
+            _status.TryRemoveStatusEffect(victim, "Muted");
+            _status.TryRemoveStatusEffect(victim, "TemporaryBlindness");
+        }
```

Comment:
This might cause issues if something else applies a longer muted and temporary blindness status but I think it's fine for now